.travis.yml

```diff
@@ -1,6 +1,16 @@
+sudo: required
+
 language: ruby
 rvm:
   - 2.0
```

Comment:
I wonder (and this is nothing for this PR specifically) if we should drop explicit support for 2.0 (and perhaps even 2.1)